include/settings/settings.h

```diff
@@ -120,6 +131,15 @@ int settings_register(struct settings_handler *cf);
  */
 int settings_load(void);
 
+/**
+ * Load limited set of serialized items from registered persistence sources.
+ * Handlers for serialized item subtrees registered earlier will be called for
+ * encountered values that belong to the subtree
+ *
```

Comment:
Param documentation missed.

Reply:
@param[in] subtree name of the subtree to be loaded.